src/vs/workbench/electron-browser/integration.ts

```diff
@@ -134,12 +134,14 @@ export class ElectronIntegration {
 		ipc.on('vscode:enterFullScreen', (event) => {
 			this.partService.joinCreation().then(() => {
 				this.partService.addClass('fullscreen');
+				this.partService.layout();
```

Comment:
I thought it wouldn't do any harm on both context's, but I'll wrap it, makes it easier to see what is cause by the inline titlebar too 👍